### PR TITLE
Add mini development notes

### PR DESCRIPTION
**Pull request by dmey**
This is just something I found/find useful and much lighter than the previous iteration. If you find it as well please mere it in otherwise will keep in in my local copy.

### CODE REVIEW

File: docs/src/development.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Development notes
+
+To run tests:
```
Comment (navidcy) [inline docs/src/development.md:3]:
```suggestion
To run tests, from the path of your local clone of the repository run:
```

File: docs/src/development.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+```
+julia --project=. -e 'import Pkg; Pkg.test()'
```
Comment (navidcy) [inline docs/src/development.md:5]: simpler
```suggestion
julia --project -e 'import Pkg; Pkg.test()'
```

File: docs/src/development.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+```
+
+To install dependencies:
+
+```
+julia --project=. -e 'import Pkg; Pkg.instantiate()`
```
Comment (navidcy) [inline docs/src/development.md:11]:
```suggestion
julia --project -e 'import Pkg; Pkg.instantiate()`
```

File: docs/src/development.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+```
+
+then:
+```
+julia --project=.
```
Comment (navidcy) [inline docs/src/development.md:16]:
```suggestion
julia --project
```

Comment (navidcy) [inline docs/src/development.md:16]:
```suggestion
```

Perhaps simply delete these lines? They are ambiguous... "Then" what? The dependencies are already installed from the previous step.

File: docs/src/development.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+```
+
+To generate docs:
+
+```
+julia --project=docs -e 'import Pkg; Pkg.develop(path="."); Pkg.instantiate()'
+julia --project=docs docs/make.jl
+```
```
Comment (navidcy) [inline docs/src/development.md:24]:
```suggestion
```

If the docs are built successfully, you view them by opening `docs/build/index.html` in your favorite browser.
```

**Comment by milankl** [comment]
@dmey do you want de-draft this pr or add further commits to it? Trying to clean up things!

**Review by navidcy — APPROVED** [review]

**Comment by navidcy** [comment]
I think this can be merged and further tweaked in the future when needed.

**Pull request merged**